develop/bootstrap.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+CLUSTER_NAME="dvo-local"
+PROMETHEUS_NAMESPACE="prom"
+GRAFANA_NAMESPACE="graf"
+DEVELOPMENT_NAMESPACE="check"
+DVO_NAMESPACE="deployment-validation-operator"
+
+INSTALL_PROMETHEUS="true"
+INSTALL_PROMETHEUS_OPERATOR="false"
+INSTALL_GRAFANA="false"
+
+k3d cluster list | grep ${CLUSTER_NAME} &> /dev/null
+SUCCESS=$?
+if [ $SUCCESS -eq 1 ]
+then
+    echo "Creating ${CLUSTER_NAME} k3d cluster"
+    k3d cluster create $CLUSTER_NAME -p "8080:30080@server:0" -p "8081:30081@server:0" --registry-create $CLUSTER_NAME-registry:0.0.0.0:5432 --registry-config registries.yaml
+    SUCCESS=$?
+    if [ $SUCCESS -eq 1 ]
+    then
+        echo "Error creating k3d cluster"
+        exit 1
+    fi
+    echo ""
+fi
+
+CURRENT_CONTEXT=$(kubectl config current-context)
+if [ "$CURRENT_CONTEXT" != "k3d-$CLUSTER_NAME" ]
+then
+    kubectl config use-context $CLUSTER_NAME
+    SUCCESS=$?
+    if [ $SUCCESS -eq 1 ]
+    then
+        echo "Error selecting k3d kubernetes context"
+        exit 1
+    fi
+fi
+
+if [ "$INSTALL_PROMETHEUS" == "true" ]
+then
+    echo "Installing prometheus"
+    echo ""
+    helm install -n $PROMETHEUS_NAMESPACE --create-namespace --set server.service.type=NodePort --set server.service.nodePort=30080 prom prometheus-community/prometheus &> /dev/null
```

Comment:
Can we do these actions without growing a dep on helm?  We'll already have deps on oc, kubectl, tilt, k3d at the very least.  I'd like the dep list to be as small as possible.

Reply:
NM.  Helm is installed and managed later in the script.